syne_tune/optimizer/baselines.py

```diff
@@ -465,10 +465,46 @@ def __init__(
         )
 
 
+class HEBO(SingleObjectiveScheduler):
+    """
+    HEBO
```

Comment:
Can you add a reference to the original paper? Also mention that we wrap the optuna code